arm-datalake-store/filesystem/2016-11-01/swagger/filesystem.json

```diff
@@ -529,14 +527,11 @@
             "type": "string",
             "description": "The constant value for the operation.",
             "enum": [
-              "GETFILESTATUS"
+              "MSGETFILESTATUS"
```

Comment:
No because they behave the same with some additional properties returned in MSGETFILESTATUS. The caller doesn't know the OP query param is different, they only know that they are getting more stuff back (nothing else is renamed).